File: DTO/Base/ModelBase.cs
namespace DTO.Base
{
    public class ModelBase
    {
        public Guid Id { get; set; }
        public DateTime CreatedAt { get; set; }
        public string? CreatedBy { get; set; }
        public DateTime UpdatedAt { get; set; }
        public string? UpdatedBy { get; set; }
        public bool IsEdit { get; set; } = false;
        public bool IsActived { get; set; } = true;
        public int Sort { get; set; }
    }
}

File: CLIENT/src/app/models/base/modelBase.ts
export class ModelBase {
  id: string | null = null;
  createdAt: Date | null = null;
  createdBy: string | null = null;
  updatedAt: Date | null = null;
  updatedBy: string | null = null;
  isEdit: boolean = false;
  isActived: boolean = true;
  sort: number | null = null;
}
